index.js

```diff
@@ -424,6 +425,21 @@ class Hyperbee {
   close () {
     return this.feed.close()
   }
+
+  static async isHyperbee (core) {
```

Comment:
add opts here and forward those opts to the get below. then you can do
 `isHyperbee(core, { wait: false })` etc